python_modules/dagster/dagster/_grpc/server.py

```diff
@@ -409,7 +451,9 @@ def ReloadCode(
 
     def Ping(self, request, _context: grpc.ServicerContext) -> api_pb2.PingReply:
         echo = request.echo
-        return api_pb2.PingReply(echo=echo)
+        return api_pb2.PingReply(
+            echo=echo, serialized_server_health_metadata=json.dumps(_UTILIZATION_METRICS)
```

Comment:
I think we should be a transform step between what raw data we are trakcing and what we send to keep the payload small if we are going to re-use ping.

Reply:
Definitely makes sense. I think to start we can just retrieve identifying information from the request params; ie sensor name

Reply:
my intuition was even less - like just `current_sensor_evals: 4`